scilpy/reconst/raw_signal.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+import numpy as np
+
+from dipy.core.sphere import Sphere
+from dipy.reconst.shm import sf_to_sh
+
+from scilpy.utils.bvec_bval_tools import (check_b0_threshold,
+                                          is_normalized_bvecs, normalize_bvecs,
```

Comment:
identify_shells comes before normalize_bvecs, not sure for the last one since it is a variable

Reply:
I changed the identify, but I couldn't find the info if variables are treated differently. Keeping as is.